frontend/sw360-portlet/src/main/resources/META-INF/resources/html/components/includes/releases/clearingDetails.jspf

```diff
@@ -0,0 +1,346 @@
+<%--
+  ~ Copyright Siemens AG, 2013-2017, 2019. Part of the SW360 Portal Project.
+  ~
+  ~ SPDX-License-Identifier: EPL-1.0
+  ~
+  ~ All rights reserved. This program and the accompanying materials
+  ~ are made available under the terms of the Eclipse Public License v1.0
+  ~ which accompanies this distribution, and is available at
+  ~ http://www.eclipse.org/legal/epl-v10.html
+--%>
+<%@ page import="org.eclipse.sw360.portal.common.FossologyConnectionHelper" %>
+
+<portlet:resourceURL var="loadSpdxLicenseInfoUrl">
+    <portlet:param name="<%=PortalConstants.ACTION%>" value='<%=PortalConstants.LOAD_SPDX_LICENSE_INFO%>'/>
+</portlet:resourceURL>
+<portlet:resourceURL var="writeSpdxLicenseInfoIntoReleaseUrl">
+    <portlet:param name="<%=PortalConstants.ACTION%>" value='<%=PortalConstants.WRITE_SPDX_LICENSE_INFO_INTO_RELEASE%>'/>
+</portlet:resourceURL>
+
+<%@include file="/html/utils/includes/fossologyClearing.jspf"%>
+
+<core_rt:set var="clearingInfo" value="${release.clearingInformation}"/>
+<core_rt:set var="FOSSOLOGY_CONNECTION_ENABLED" value="<%=FossologyConnectionHelper.getInstance().isFossologyConnectionEnabled()%>"/>
+
+<table class="table table-bordered" id="spdxAttachments"
+        data-load-spdx-license-info-url="<%=loadSpdxLicenseInfoUrl%>"
+        data-write-spdx-license-info-into-release-url="<%=writeSpdxLicenseInfoIntoReleaseUrl%>"
+        data-release-id-parameter-name="<%=PortalConstants.RELEASE_ID%>"
+        data-attachment-id-parameter-name="<%=PortalConstants.ATTACHMENT_ID%>"
+        data-spdx-license-info-parameter-name="<%=PortalConstants.SPDX_LICENSE_INFO%>">
+    <colgroup>
+        <col style="width: 60%;"/>
+        <col style="width: 5.1rem;"/>
+        <col style="width: 40%;"/>
```

Comment:
Only the action column is fixed because it is known that it will contain one button with predefined text. So not sure how an improvement could look like?